app/browser/tabs.js

```diff
@@ -302,7 +302,8 @@ const fixDisplayURL = (navigationEntry, controller) => {
     navigationEntry.virtualURL = getSourceAboutUrl(navigationEntry.virtualURL)
   }
 
-  if (isIntermediateAboutPage(navigationEntry.virtualURL)) {
+  if (isIntermediateAboutPage(navigationEntry.virtualURL) &&
+    navigationEntry.virtualURL.indexOf('about:safebrowsing#') === -1) {
```

Comment:
maybe not because those errors fire after the page has started loading already but about:safebrowsing fires before